flask_restplus/marshalling.py

```diff
@@ -44,14 +50,22 @@ def make(cls):
         fields = apply_mask(fields, mask, skip=True)
 
     if isinstance(data, (list, tuple)):
-        out = [marshal(d, fields) for d in data]
+        out = [marshal(d, fields, skip_none=skip_none) for d in data]
         if envelope:
             out = OrderedDict([(envelope, out)])
         return out
 
-    items = ((k, marshal(data, v) if isinstance(v, dict)
-              else make(v).output(k, data))
-             for k, v in fields.items())
```

Comment:
Thanks for the suggestion! Please help to review if the changes meet the requirement :)